internal/app/organizationConfig/helper.go

```diff
@@ -0,0 +1,19 @@
+package organizationConfig
+
+import (
+	"github.com/joshsoftware/peerly-backend/internal/pkg/dto"
+	"github.com/joshsoftware/peerly-backend/internal/repository"
+)
+
+func organizationConfigToDTO(org repository.OrganizationConfig) dto.OrganizationConfig {
+	return dto.OrganizationConfig{
+		RewardMultiplier:            org.RewardMultiplier,
+		ID:                          org.ID,
```

Comment:
correct the sequence, ID should be first always.

Reply:
updated